akka-stream/src/main/scala/akka/stream/KillSwitch.scala

```diff
@@ -89,15 +102,16 @@ object KillSwitches {
 
     override val initialAttributes = Attributes.name("breaker")
     override val shape = BidiShape(
-      Inlet[Any]("KillSwitchBidi.in1"), Outlet[Any]("KillSwitchBidi .out1"),
```

Comment:
Also run into this typo. This is a separate commit, so it can be easily removed.